lighthouse-core/config/default-config.js

```diff
@@ -21,8 +21,8 @@ const UIStrings = {
   metricGroupTitle: 'Metrics',
   /** Title of the opportunity section of the Performance category. Within this section are audits with imperative titles that suggest actions the user can take to improve the loading performance of their web page. 'Suggestion'/'Optimization'/'Recommendation' are reasonable synonyms for 'opportunity' in this case. */
   loadOpportunitiesGroupTitle: 'Opportunities',
-  /** Description of the opportunity section of the Performance category. 'Optimizations' could also be 'recommendations' or 'suggestions'. Within this section are audits with imperative titles that suggest actions the user can take to improve the loading performance of their web page. */
-  loadOpportunitiesGroupDescription: 'These optimizations can speed up your page load.',
+  /** Description of the opportunity section of the Performance category. 'Suggestions' could also be 'recommendations'. Within this section are audits with imperative titles that suggest actions the user can take to improve the loading performance of their web page. */
+  loadOpportunitiesGroupDescription: 'These suggestions can help your page load faster. They don\'t [directly affect](https://github.com/GoogleChrome/lighthouse/blob/master/docs/scoring.md#how-are-the-scores-weighted) the Performance score.',
```

Comment:
maybe we should be sha (or at least version) tagging these links if they're going to live in a report forever?

Reply:
> ~maybe we should be sha (or at least version) tagging these links if they're going to live in a report forever?~
> 
> ignore this I just read the "links are in progress" for the web fundamentals update :)

actually, maybe we won't wait for that. At worst, 5.2 users will get a special easter egg doc link. Added the hash as suggested, though :)

Reply:
> ~maybe we should be sha (or at least version) tagging these links if they're going to live in a report forever?~
> 
> ignore this I just read the "links are in progress" for the web fundamentals update :)

actually, maybe we won't wait for that. At worst, 5.2 users will get a special easter egg doc link. Added the hash as suggested, though :)